### DISCUSSION TITLE
DOC: Suggest using Typer version instead of "Python 3.6+ vs Python 3.6+ Non-Annotated" to distinguish the documentation between v0.9.0 and <=v0.8.0

### DISCUSSION THREAD

**Discussion by LYSingD** [fastapi/typer#611] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

![image](https://github.com/tiangolo/typer/assets/20653350/bf878337-8845-4cff-acaf-c19f259fe3f9)

In the documentation of Option & Arugment, the context "Python 3.6+ vs Python 3.6+ Non-Annotated" in the tabs is not intuitive to show there was a version upgrade. Our team is still using version `0.7.0`, and it took us a while to realize we should look up the content in "Python 3.6+ Non-Annotated."

Would renaming the context in the tabs to "latest vs <= v0.8.0" or something like that also make sense to you? 

Thanks!

### Operating System

Linux

### Typer Version

0.7.0

### Python Version

3.11.3

**Comment by YuriiMotov** [comment]
I guess this is outdated now)
